### PR TITLE
Metalk8s UI: add intergration test of create volume error scenario

### PR DESCRIPTION
**Pull request by ChengYanJin**
**Component**: ui, test

**Context**: 
This PR takes care of the following point in the original ticket: https://github.com/scality/metalk8s/issues/2878

>  Test Creation scenarios (create Volume from Volume page, from Node page, test various error cases, ...)

**Summary**:

Add two error scenarios of creating volume.
- K8s API is unavailable
- k8s API return `CreationError` for the volume just created

**Acceptance criteria**: 

---

Closes: #ISSUE_NUMBER

### CODE REVIEW

File: ui/cypress/fixtures/kubernetes/volumecreation.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+       "resourceVersion":"71330993",
+       "uid":"2ef6fd7c-c9b6-4d98-bfa7-a61245e41d15"
+    },
+    "spec":{
+       "mode":"Filesystem",
+       "nodeName":"master-0",
+       "rawBlockDevice":{
+          "devicePath":"/sddsf"
+       },
+       "storageClassName":"hyperdrive-data",
```
Comment (JBWatenbergScality) [inline ui/cypress/fixtures/kubernetes/volumecreation.json]: nit: we should rename this storage class

```suggestion
       "storageClassName":"ssd-ext4",
```

Comment (ChengYanJin) [reply ui/cypress/fixtures/kubernetes/volumecreation.json]: Fixed in 37c898b23e73789cd59fb9f486a702b4a56d4ef1

**Review by JBWatenbergScality — APPROVED** [review]

**Pull request merged**